### ISSUE TITLE
How to change/add custom property to request sent by superset?

### ISSUE THREAD

**Issue by jatinmehrotra** [forwardemail/supertest#740]
My backend relies on value which is added in request object as req.requestContext by api ateway.
In order to test this i want to add custom property to request sent by superset, how do i achieve this?

**Comment by aleksandarcrvc** [comment]
I have same requirement and just wandering is this possible?

**Comment by reinaldoacdc** [comment]
I have the same issue.

**Comment by Nide83** [comment]
is it possible to test a custom property in request object now?

**Comment by Vitomir2** [comment]
@Nide83 , I do not think so - I am currently trying also to find a way to send custom property inside the request.
